pkg/webui/console/store/middleware/logics/gateways.js

```diff
@@ -78,8 +79,24 @@ const getGatewaysLogic = createRequestLogic({
         )
       : await api.gateways.list({ page, limit }, selectors)
 
+    const entities = await Promise.all(
```

Comment:
Can you make the decoration with the status value optional via a flag in `action.meta`?
I think there will be situations where we want to avoid the extra request.